glean-core/rlb/src/dispatcher/global.rs

```diff
@@ -46,6 +50,17 @@ pub fn launch(task: impl FnOnce() + Send + 'static) {
     }
 }
 
+/// Launches a new task on the global dispatch queue with a reference to the Glean singleton.
+pub fn launch_with_glean(callback: impl FnOnce(&Glean) + Send + 'static) {
+    dispatcher::launch(|| crate::with_glean(callback));
+}
+
+/// Launches a new task on the global dispatch queue with a mutable reference to the
+/// Glean singleton.
+pub fn launch_with_glean_mut(callback: impl FnOnce(&mut Glean) + Send + 'static) {
+    launch(|| crate::with_glean_mut(callback));
+}
```

Comment:
Right now the dispatcher module has no knowledge about Glean, which is a good thing.

Let's move these functions into the top level module, e.g. [right after `with_glean_mut` here](https://github.com/mozilla/glean/blob/4445cf3a1bcec8c696264dbcc02c8aae17c355db/glean-core/rlb/src/lib.rs#L152).

Whereever we call it then we can use `crate::launch_with_glean` instead of `dispatcher::launch(...)`.

Reply:
Thanks! Just made the change and squashed the two commits into one.